### PR TITLE
add force delete user endpoint

### PR DESCRIPTION
**Pull request by cyinma**
This is to add a new endpoint under new admin service to force delete a user including the organization and resources they have.

### CODE REVIEW

**Review by johanbrandhorst — APPROVED** [review]
LGTM, but wait for others to weigh in before merging

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/organization.proto";
+import "buf/alpha/registry/v1alpha1/plugin.proto";
+import "buf/alpha/registry/v1alpha1/repository.proto";
+import "buf/alpha/registry/v1alpha1/user.proto";
+
+// UserService is the User service.
```
Comment (robbertvanginkel) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:24]: This comment seems wrong.

**Comment by robbertvanginkel** [comment]
I think this is the most reasonable way forward today given the circular import problem. I suppose because we're on `FILE` breaking we can't just move things around, even though for go it doesn't matter as much.

What I'm not 100% convinced of is saying only admins can force delete. As a user if I want to leave buf, I think forcing me to go through all of my resources individually and manually delete them is a bad experience. Maybe we don't care, because the user is leaving anyway, but I do feel that this direction makes it harder to revisit that experience.

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+service AdminService {
+  // ForceDeleteUser forces to delete a user. Resources and organizations that are
+  // solely owned by the user will also be deleted.
+  rpc ForceDeleteUser(ForceDeleteUserRequest) returns (ForceDeleteUserResponse);
```
Comment (robbertvanginkel) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:28]: I would rather see a generic "DeleteUser" with a parameter around deleting. More flexible if for whatever reason we want to let admins to "soft"(?) deletes of other users.

Comment (robbertvanginkel) [reply proto/buf/alpha/registry/v1alpha1/admin.proto:28]: Resolved offline, keeping as is.

**Review by robbertvanginkel — APPROVED** [review]

**Review by amckinney — APPROVED** [review]

**Pull request merged**